e2e/ganache/index.js

```diff
@@ -0,0 +1,41 @@
+const ganache = require('ganache-cli')
+const getPort = require('get-port')
+
+class Ganache {
+  async start (options) {
+    const port = options.port || await getPort()
+    this._server = ganache.server(options)
+    return new Promise((resolve, reject) => {
+      this._server.listen(port, (err, blockchain) => {
+        if (err) {
+          return reject(err)
+        }
+
+        resolve({
+          ...blockchain,
+          port,
+        })
+      })
+    })
+  }
+
+  async quit () {
+    if (!this._server) {
+      return Promise.reject(new Error('Server not running yet'))
```

Comment:
This is equivalent to `throw new Error('Server not running yet')` - it's just being wrapped in an additional Promise.